steps/appium-server/0.0.1/step.yml

```diff
@@ -0,0 +1,47 @@
+title: Appium Server
+summary: |
+  Install and Run Appium Server
+description: |
+  Install and Run Appium Server on Bitrise to be able to run our mobile automated tests
+website: https://github.com/moatazeldebsy/bitrise-step-appium-server
+source_code_url: https://github.com/moatazeldebsy/bitrise-step-appium-server
+support_url: https://github.com/moatazeldebsy/bitrise-step-appium-server/issues
+published_at: 2021-08-16T00:23:58.547135+02:00
+source:
+  git: https://github.com/moatazeldebsy/bitrise-step-appium-server.git
+  commit: 9267727bfaa73a7eaccfb1b976d4aebeff6d3fc0
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- appium_version: Default Value - you can leave this empty if you want to
```

Comment:
One of the value options should be the default value (`v1.21.0` or `v1.20.2`) for the input.